apps/web/src/lib/elastic/client.ts

```diff
@@ -1,9 +1,10 @@
+import env from '$/env'
 import { Client } from '@elastic/elasticsearch'
 
 export default new Client({
   node: process.env.ELASTIC_URL,
```

Comment:
What's `ELASTIC_URL`?

Reply:
Linter fail. We have to setup CI. I'll do tomorrow
![image](https://github.com/latitude-dev/latitude-llm/assets/49499/9b56a40a-5ba6-4e43-9d8f-02cb0ecb3cda)

Reply:
Put defaults values in [env.example](https://github.com/latitude-dev/latitude-llm/blob/main/apps/web/env.example) so we can use it